src/ResourceManager/Insights/Commands.Insights/help/Add-AzureRmLogProfile.md

```diff
@@ -106,7 +117,11 @@ Accept wildcard characters: False
 ```
 
 ### -StorageAccountId
-Specifies the ID of the Storage account.
+Specifies the ID of the Storage account. ID is the fully qualified Resource ID of the storage account for example  
+```yaml
```

Comment:
@cormacpayne if I have removed the yaml from around the block, but if I remove the ``` then command doesn't get into a block. Also I haven't added these blocks, these were present in the original version. I am just adding content to them as it's mostly empty.